server/src/main/java/org/elasticsearch/plugins/ActionPlugin.java

```diff
@@ -64,6 +65,7 @@ default List<ActionFilter> getActionFilters() {
      */
     default List<RestHandler> getRestHandlers(
```

Comment:
Similar to `createComponents`, it may be worth introducing a containment interface for these method parameters